src/Spout/Reader/ODS/RowIterator.php

```diff
@@ -161,7 +196,9 @@ public function next()
             throw new IOException("The sheet's data cannot be read. [{$exception->getMessage()}]");
         }
 
-        $this->rowDataBuffer = $rowData;
+        for ($i = 0; $i < $numRowsRepeated; ++$i) {
```

Comment:
:+1: for @adrilo 
Plus, the `next()` call will return more than one row as if the iterator had read multiples rows.